doc/docs/download_cfd.md

```diff
@@ -10,35 +10,35 @@ See the [Launcher overview](overview_cfd.md) and [Installing nRF Connect for Des
 
 ## Additional requirements
 
-Running nRF Connect for Desktop has the following additional requirements.
+Running nRF Connect for Desktop has the following additional requirements:
 
-### SEGGER J-Link driver
+- **SEGGER J-Link** - Required on all platforms.
 
-Required on all platforms.
+    - On Windows, the driver comes bundled with nRF Connect for Desktop.
+    - On macOS and Linux, you must install the driver manually. Download the installer for your platform from [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
 
-On Windows, the driver comes bundled with nRF Connect for Desktop.
+     If you don't install the J-Link driver, running the apps will be very limited and you will get error messages `CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
 
-On macOS and Linux, you must install the driver manually.
-Download the installer for your platform from [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
-If you don't install the J-Link driver, running the apps will be very limited and you will get error messages `CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
+- For Linux:
 
-### Installing libusb-1.0-0 and nrf-udev on Linux
+    - **libusb-1.0-0** - Usually comes installed with Ubuntu and you can install it with the following command:
 
-libusb-1.0-0 usually comes installed with Ubuntu.
+        ```
+        sudo apt install libusb-1.0-0
+        ```
 
-nrf-udev can be installed by downloading a DEB file from the [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev) project repository.
+    - **nrf-udev** - The _udev_ rules are required to access Nordic USB devices with correct permissions. Download the latest DEB file from [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev) and run the following command to install nrf-udev:
 
-The _udev_ rules are required to access Nordic USB devices with correct permissions.
-You need to set up these rules only once.
+        ```
+        sudo dpkg -i nrf-udev_1.0.1-all.deb
+        ```
 
-Complete the following steps:
+    - **libfuse2** - Required for Ubuntu v22.04 and above to run AppImage applications.
+      You can install it with the following command:
 
-1. Download the latest DEB file from [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev).
-2. Run the following command to install nrf-udev:
-
-    ```
-    sudo dpkg -i nrf-udev_1.0.1-all.deb
-    ```
+        ```
+        sudo apt install libfuse2
```

Comment:
```suggestion
        sudo add-apt-repository universe
        sudo apt install libfuse2
```

not sure this will work without updating repository. Maybe yes, maybe not 

----
@boundlesscalm maybe you can confirm that Ubuntu 20.04 (minimal installation) do not require fuse and it will be pre-installed?

One more concern / question: seems minimal installation of the Ubuntu 22.04 does not have libfuse2, but full install will do, making this step optional..

Reply:
> not sure this will work without updating repository. Maybe yes, maybe not

I need more than "maybe yes, maybe not" to make this change.

Does this mean the installation command I've provided is incorrect?

Reply:
Not sure about Ubuntu 20.04.
@greg-fer The command is correct. It may just happen that it is already installed for some users.
Sadly I don't know if the universe repo is required but I'll trust @KievDevel on this.

Reply:
If not sure, then I will keep the current wording (22.04 onwards).

Reply:
To handle all the possible cases updated suggested commands list (also discussed with @chunfantasy)